src/Host/Client/Impl/Session/RSession.cs

```diff
@@ -560,6 +568,7 @@ public Task StartHostAsync(RHostStartupInfo startupInfo, IRSessionCallback callb
             Prompt = GetDefaultPrompt(prompt);
             MaxLength = len;
 
+            IsProcessing = contexts.Count != 1;
```

Comment:
I'm not sure that it is enough. If there are any inputs in the queue, `IsProcessing` may be switched back to `true` after index has started update. Also, is it possible that any of our evaluation commands also blocks dll's?